src/DynamoCore/Graph/Workspaces/WorkspaceModel.cs

```diff
@@ -2530,7 +2530,7 @@ private void LoadNotesFromAnnotations(IEnumerable<ExtraAnnotationViewInfo> annot
                 var text = annotationViewInfo.Title;
 
                 var pinnedNode = this.Nodes.
-                    FirstOrDefault(x => x.GUID.ToString("N") == annotationViewInfo.PinnedNode);
+                    FirstOrDefault(x => x.GUID.ToString() == annotationViewInfo.PinnedNode);
```

Comment:
Curious why we remove the formatting? Did you find it does not matter in this case and the default format is "N" anyway?

Reply:
I honestly don't know why the formatting was there to start with, and how it was functioning before. With the formatting, the code was breaking because it wasn't finding the necessary Node the Note was pinned to.

Reply:
Reverted back after the update. I believe that the extra Guid assignment I was doing was the reason the formatting didn't function. In any case, this change is now removed.